core-services/image-mirroring/supplemental-ci-images/README.md

```diff
@@ -0,0 +1 @@
+The mapping files in this folder are replaced with the mapping in the configuration file in the core-services/image-mirroring/_config.yaml.
```

Comment:
Could you please add the link https://docs.ci.openshift.org/docs/how-tos/external-images/ in the readme?